gfeloqua.class.php

```diff
@@ -1410,6 +1410,9 @@ function log_entry_notes( $entry_id, $form_id = NULL ){
 		$notes = array_merge( $notes, $new_notes );
 
 		gform_update_meta( $entry_id, GFELOQUA_OPT_PREFIX . 'notes', $notes, $form_id );
+
+		// Added hook to allow custom error logging.
+		do_action( 'gravityforms_eloqua_log_entry_notes', $this->debugger, $entry_id, $form_id );
```

Comment:
I have address this in my latest commit (e886b2d6b18e2ebcfe0b523233285eec19c427e3)